x/ethbridge/genesis_test.go

```diff
@@ -15,7 +15,6 @@ func TestExportGenesis(t *testing.T) {
 	// Generate State
 	receiver := CreateState(ctx, keeper, t)
 	state := ethbridge.ExportGenesis(ctx, keeper)
-	// assert.Equal(t, len(state.PeggyTokens), tokenscount)
```

Comment:
Why was this assertion removed?

Reply:
no PeggyTokens in state anymore, and assertation already commented.